governance.md

```diff
@@ -202,6 +212,15 @@ this reinstatement.
 Emeritus Council members will be listed on the project website, acknowledging
 the period during which they were active in the Council.
 
+Currently, the Eremitus Council members are:
```

Comment:
If someone has information on the period of activity mentioned above, please update this list.